### DISCUSSION TITLE
❌ Transfer support

### DISCUSSION THREAD

**Discussion by skull-squadron** [isontheline/pro.webssh.net#704] [Ideas]
I don't want to download one file at a time. This is trivial. I want to move or copy an entire directory from/to Google Drive, smb/cifs/afp, iCloud, or another type of storage iOS supports OR another host. Otherwise, there's no point having a file transfer capability.

**Comment by isontheline** [comment]
I understand but WebSSH transfer won't support these features in a near time.
There are plenty apps better than WebSSH about file transfer on the App Store.
